datastream/json.py

```diff
@@ -0,0 +1,11 @@
+import json
```

Comment:
Perhaps this belongs in `tools`?

Reply:
I thought about this too but thought it might be ugly but maybe it's fine. I'll change it